packages/appstore/src/types/wallet-steps.types.ts

```diff
@@ -1,7 +1,12 @@
-export type TStepProps = {
+export type TWalletsIntro = {
     title: string;
     description: string;
     bullets: string[];
     eu_user?: boolean;
-    icon?: React.ReactNode;
+    image?: React.ReactNode;
+};
+
+export type TReadyToUpgradeWallets = {
```

Comment:
@aizad-deriv This is only used in the `ReadyToUpgradeWallets` component, No need to put it in a different file.

Reply:
Done 😃 👍🏼